src/telemetry/events/__init__.py

```
@@ -0,0 +1,189 @@
+    "AgentToolConclusionsDeletedEvent",
+    "AgentToolPeerCardUpdatedEvent",
+    "AgentToolSummaryCreatedEvent",
+    # Reconciliation events
+    "SyncVectorsCompletedEvent",
+    "CleanupStaleItemsCompletedEvent",
+    # Deletion events
+    "DeletionCompletedEvent",
+    # Lifecycle
+    "initialize_telemetry_events",
+    "shutdown_telemetry_events",
+]
```

Ruff (0.14.13)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)